project/Dependencies.scala

```diff
@@ -4,7 +4,7 @@ object Dependencies {
 
   val play = "com.typesafe.play" %% "play" % "2.6.7"
   val identityCookie = "com.gu.identity" %% "identity-cookie" % "3.80"
-  val identityTestUsers = "com.gu" %% "identity-test-users" % "0.6"
+  val identityTestUsers = "com.gu" %% "identity-test-users" % "0.7"
```

Comment:
I think the PR title should be `identity-test-users` not `identity-cookie`, based on this?

Reply:
Oops.  Thanks.